cpu/msp430fxyz/periph/gpio.c

```diff
@@ -81,15 +81,46 @@ static int _ctx(gpio_t pin)
 int gpio_init(gpio_t pin, gpio_dir_t dir, gpio_pp_t pullup)
 {
     msp_port_t *port = _port(pin);
-    /* check if port is valid and pull resistor are valid */
-    if ((port == NULL) || (pullup != GPIO_NOPULL)) {
+    /* check if port is valid */
+    if (port == NULL)
         return -1;
```

Comment:
Please conform with the coding conventions (add braces in this case).
Uncrustify should be able to help, the wiki otherwise.